### ISSUE TITLE
Reihenfolge der Downloads ändern

### ISSUE THREAD

**Issue by scytho** [mediathekview/MediathekView#293]
Hallo,

ich habe ab und zu eine etwas größere Menge an Sendungen in der Downloadliste und würde MV gerne irgendwie mitteilen, daß es die Sendungen nicht in der Reihenfolge herunterladen soll, in der sie in der Liste gelandet sind (Spalte »Nr«), sondern die ältesten (Spalten »Datum« und »Zeit«) zuerst, weil die zuerst depubliziert werden. Geht das jetzt schon irgendwie, oder ist das ein Fall für MV14?

**Comment by zxsd** [comment]
Das [MV-Forum](https://forum.mediathekview.de/) wäre künftig die geeignete Anlaufstelle, um eine schnelle Antwort zu bekommen.

Außer Du schon viel mit den Spaltennamen 'rumgespielt hast, brauchst Du nur die bedeutsamen Spaltennamen zu klicken (1x oder 2x, hinauf oder herunter).  Sobald die Reihenfolge stimmt, gib die Sendungen zum Herunterladen frei!

**Comment by scytho** [comment]
Ja, das habe ich inzwischen gemerkt. Allerdings benutzt MV die vorgegebene Reihenfolge, wie du andeutest, erst, wenn man alle Downloads anhält und dann wieder startet, oder eben wenn man sie ungestartet zur Liste hinzufügt und erst dann startet, wenn alle in der Liste sind. Wenn man danach Downloads hinzufügt oder wenn man sie von Anfang an so hinzufügt, daß sie sofort gestartet werden, bleibt MV stur bei der Hinzufügereihenfolge, auch wenn die Spaltensortierung etwas anderes vorgibt. Das würde ich (nachwievor) als Bug ansehen, deswegen lasse ich es erstmal hier.

**Comment by derreisende77** [comment]
dies ist durch anordnen der downloads via drag and drop möglich.
